### ISSUE TITLE
Canvas don't log and text hoover

### ISSUE THREAD

**Issue by Dracks977** [browserspy/browserspy#7]
## What are you using - recorder, player, etc - and version (if known)?
v0.4.1 recorder and player

## Expected behaviour
see when the text is highlight and see the canvas

## Actual behaviour
the two do not display or record

## Steps to reproduce
I added you to the project

**Comment by riggerthegeek** [comment]
Canvas support will be added, but my current plan is for post-MVP.

By text highlight, do you mean when the user highlights text with mouse/keyboard (eg, when pressing CMD/CTRL + A)? This might be very tricky to achieve as there's no actual change to the DOM - certainly not impossible, but again likely a future enhancement after MVP.

**Comment by Dracks977** [comment]
yep, when ctrl A or with mouse ^^

**Comment by riggerthegeek** [comment]
Yeah, that'll be after MVP (unless of course there's a massive amount of +1's added to this issue).

Thanks for raising the issue

**Comment by riggerthegeek** [comment]
Yeah, that'll be after MVP (unless of course there's a massive amount of +1's added to this issue).

Thanks for raising the issue

**Comment by Dracks977** [comment]
np, nice concept by the way
